packages/disploy/src/structs/Message.ts

```diff
@@ -160,9 +160,12 @@ export class Message extends Base {
 	 * The stickers in this message.
 	 */
 	public readonly stickerItems: APIStickerItem[];
-
+	/**
+	 * The raw data of the message.
+	 */
+	declare public raw: APIMessage;
 	public constructor(app: App, raw: APIMessage) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.